libs/core/langchain_core/schema/docstore.py

```diff
@@ -0,0 +1,29 @@
+"""Interface to access to place that stores documents."""
+from abc import ABC, abstractmethod
+from typing import Dict, List, Union
+
+from langchain_core.documents import Document
+
+
+class DocstoreInterface(ABC):
```

Comment:
This interface should be deprecated from langchain -- It's not a doc-store

Reply:
I'm in favor of removing the obsolete classes.